packages/core/package.json

```diff
@@ -1,27 +1,31 @@
 {
   "name": "@cornerstonejs/core",
-  "version": "1.82.5",
+  "version": "2.0.0-beta.19",
   "description": "",
-  "main": "src/index.ts",
-  "types": "dist/types/index.d.ts",
+  "main": "dist/umd/index.js",
+  "types": "dist/esm/index.d.ts",
   "module": "dist/esm/index.js",
   "repository": "https://github.com/cornerstonejs/cornerstone3D",
   "files": [
-    "dist/",
-    "src/"
+    "dist/"
   ],
   "directories": {
     "test": "test"
   },
   "sideEffects": false,
+  "exports": {
+    ".": {
```

Comment:
You might consider including: "./types": "./dist/esm/types/index.d.ts"  so that you can do:
import type { IViewport } from '@cornerstonejs/core/types';
in another module - it is really handy.